Assets/Scripts/SlipTime/SlipTimeAdherent.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+namespace SlipTime
+{
+    /// <summary>
+    /// An object that is affected by SlipTime
+    /// </summary>
+    public class SlipTimeAdherent : MonoBehaviour
```

Comment:
This needs to be a full class instead of an interface so that we can look it up as a Unity object in the `SlipTimeManager`.

Reply:
Ok, sounds good.